build/build-image/cross/Dockerfile

```diff
@@ -72,7 +72,7 @@ RUN mkdir $TMPDIR \
             github.com/jteeuwen/go-bindata/go-bindata
 
 # Download and symlink etcd. We need this for our integration tests.
-RUN export ETCD_VERSION=v3.0.4; \
+RUN export ETCD_VERSION=v2.3.7; \
```

Comment:
#32245 is updating the cross-build image to v1.6.3-4 with etcd 3.0.4. You'll probably need to update cross/VERSION to 1.6.3-5 and push a new version.